cluster-autoscaler/cloudprovider/gce/templates.go

```diff
@@ -531,6 +536,40 @@ func extractOperatingSystemDistributionFromImageType(imageType string) Operating
 	}
 }
 
+// SystemArchitecture denotes distribution of the System Architecture used by nodes coming from node group
+type SystemArchitecture string
+
+const (
+	// UnknownArch is used if the Architecture is Unknown
+	UnknownArch SystemArchitecture = ""
+	// Amd64 is used if the Architecture is x86_64
+	Amd64 SystemArchitecture = "amd64"
+	// Arm64 is used if the Architecture is ARM
+	Arm64 SystemArchitecture = "arm64"
+	// DefaultArch is used if the Architecture is used as a fallback if not passed by AUTOSCALER_ENV_VARS
+	DefaultArch SystemArchitecture = Amd64
```

Comment:
`DefaultArch` currently lives in cloudprovider/utils. It isn't really GCE-specific, so why move it here?

Reply:
It is similar to `DefaultOS`. We were using common CA defaults. But since GCE now explicitly support arch, it seems appropriate to use cloudprovider-specific default value (even if the values are the same). We now have cloudspecific ability to control default as well.

This is also similar impl to OS, distribution, image-type.

Reply:
Ok, I buy the consistency argument, let's keep it here for now. I'd consider refactoring non-GCP-specific parts (OS, arch) to some common dir in the future though.